agent/xds/listeners_ingress.go

```diff
@@ -259,8 +329,15 @@ func makeSDSOverrideFilterChains(cfgSnap *proxycfg.ConfigSnapshot,
 			return nil, err
 		}
 
+		// TODO: Merge with the TLS config from the listener
+		// Is this necessary?
+		// tlsCfg, err := resolveGatewayServiceTLSConfig(listenerCfg.TLS, *svc.TLS)
+		// if err != nil {
+		// 	return nil, err
+		// }
```

Comment:
This is currently unimplemented, but it looked like `makeCommonTLSContextFromGatewayServiceTLSConfig` only uses the gateway service SDS config rather than inheriting/merging from the listener (as the listener does from the gateway) - was that intentional or is this needed?

Reply:
Ended up removing this since it seemed unnecessary.